src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -1786,15 +1787,14 @@ private static void ensureDetailedGHPullRequest(
       GHPullRequest pr, TaskListener listener, GitHub github, GHRepository ghRepository)
       throws IOException, InterruptedException {
     final long sleep = 1000;
-    int retryCountdown = 4;
+    int retryCountdown = SystemProperties.getInteger(GitHubSCMSource.class.getName() + ".mergeableStatusRetries", Integer.valueOf(4));
 
     while (pr.getMergeable() == null && retryCountdown > 1) {
       listener
           .getLogger()
           .format(
               "Waiting for GitHub to create a merge commit for pull request %d.  Retrying %d more times...%n",
```

Comment:
this just cleans up the incorrect logging - as it would print "retrying 2 more times" but only retry once more as `2` became `1` after printing and `1` is not more than `1` so would not enter the loop.